### PR TITLE
[Term Entry] JavaScript Object Function: Object.getPrototypeOf()

### PR DESCRIPTION
**Pull request by phuongtrieu97coder**
### Description

-Added the Object.getPrototypeOf() term for JavaScript
-Closes Issue #4063 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@phuongtrieu97coder Nice Work!! I have mentioned some changes, please make them. Ping me when you are done with the changes.

**Comment by phuongtrieu97coder** [comment]
@avdhoottt I've finished my review with all my changes. Please proceed to your next review.

**Review by avdhoottt — APPROVED** [review]
PR is ready for the second review.

File: content/javascript/concepts/objects/terms/getPrototypeOf/getPrototypeOf.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+Title: 'Object.getPrototypeOf()'
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/objects/terms/getPrototypeOf/getPrototypeOf.md:2]:
```suggestion
Title: '.getPrototypeOf()'
```

File: content/javascript/concepts/objects/terms/getPrototypeOf/getPrototypeOf.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+Description: 'Returns the prototype value of the specified object.'
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/objects/terms/getPrototypeOf/getPrototypeOf.md:3]:
```suggestion
Description: 'Returns the prototype value of the specified object'
```

File: content/javascript/concepts/objects/terms/getPrototypeOf/getPrototypeOf.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Tags:
+  - 'Properties'
+  - 'Objects'
+  - 'Functions'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/create-a-back-end-app-with-javascript'
+---
+
+The **`Object.getPrototypeOf()`** method is a static method that returns the prototype (the value of the internal [[Prototype]] property) of the specified object.
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/objects/terms/getPrototypeOf/getPrototypeOf.md:16]:
```suggestion
The **`.getPrototypeOf()`** method is a static method that returns the prototype (the value of the internal [[Prototype]] property) of the specified object.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few changes from my side

**Comment by phuongtrieu97coder** [comment]
@dakshdeepHERE  I've finished my review with all my changes. Please proceed to your next review.

**Review by dakshdeepHERE — APPROVED** [review]
@phuongtrieu97coder looks good to merge 🚀

**Pull request merged**